### PR TITLE
fix(docs): fix the docs on the azure plugin

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/azure/AbstractConnectionInterface.java [hunk 1]
```diff
@@ -7,8 +7,7 @@
 
 public interface AbstractConnectionInterface {
     @Schema(
-        title = "The blob service endpoint.",
-        description = "additionally parses it for information (SAS token)"
```
Comment (Skraye) [inline src/main/java/io/kestra/plugin/azure/AbstractConnectionInterface.java:11]: Why did you remove the description, was it wrong ?

Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/azure/AbstractConnectionInterface.java:11]: The description is unclear. Does this mean that the endpoint gets parsed to fetch the SAS token from it?

Comment (Skraye) [reply src/main/java/io/kestra/plugin/azure/AbstractConnectionInterface.java:11]: Didn't find anything similar in the code, let's merge it like this

**Review by Skraye — APPROVED** [review]
LGTM
